test_files/unique_symbol/unique_symbol.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @fileoverview Check the type generated when using a builtin symbol as
+ * a computed property.
+ */
+
+// Note: this declaration is part of the TS standard library, but it doesn't
+// hurt to restate it here, and the test suite runner doesn't include that
+// library.
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol;
+  }
+}
+
+// This test is verifying the type of this expression, which ultimately
+// refers to some TypeScript internal __@observeable thing.
+export let x: {[Symbol.observable]: string}|null = null;
```

Comment:
can you give this one more property? That'd make it more obvious to me that it's only filtering the bad ones.

By the way, I think this also fixes another problem:

```
const x: {'quoted-thing': string}: {...};
```

Maybe add a test for that so we don't regress to just specially treating symbols? In that case, I think it'd be nice to call this test folder something like non identifier properties or so.

Reply:
Done